data/remnant/remnant 2 cognizance.txt

```diff
@@ -1360,11 +1367,9 @@ mission "Remnant: Cognizance 32"
 	description "Chilia has asked you to pick up 58 tons of refined fuel from <planet> by <date>."
 	source "Viminal"
 	destination "Clink"
-	blocked "Chilia and Taely look excited about something, but when you inquire, they shake their heads and say you need to have earned their capital license before they can tell you more."
 	deadline 19
 	to offer
 		has "Remnant: Cognizance 31: done"
-	to accept
```

Comment:
Does this not make it so that the mission won't even trigger at all if they don't have the capital license? 

This was intentionally setup this way to block people who have lost their capital license, and to tell them that this is what is happening. (otherwise we'd just get everyone who has lost their license posting "I'm stuck and this mission won't trigger" posts. with this `to accept` and `blocked` sections, players can put things together for themselves and realize that betraying the Remnant and losing their capital license has in fact cost them the ability to continue the campaign.

Reply:
I'm having difficulty finding anything in the game that will actually take away the player's remnant capital license.

Regardless, using a blocked message to indicate that the player is permanently locked out of continuing these missions because of something they did before doesn't seem very good. That message will appear every time the player comes here and seems to imply that they need to do something to meet the mission requirements (which is what every other blocked message in the game does) when, in fact, there's nothing they can do short of editing the files.